extensions/data-plane/data-plane-s3/src/main/java/org/eclipse/dataspaceconnector/aws/dataplane/s3/S3DataSinkFactory.java

```diff
@@ -42,6 +42,8 @@
 
 public class S3DataSinkFactory implements DataSinkFactory {
 
+    private static final int CHUNK_SIZE_IN_BYTES = 1024 * 1024 * 500; // 500MB chunk size
```

Comment:
Is this configurable?

Reply:
Unfortunately, not for now.

Reply:
Can you please make it so, it is an easy addition?

Reply:
@jimmarino would it be ok to have it implemented in another PR? so now we can merge this PR and the #1881

Reply:
sure